.github/workflows/tgg-unit-tests.yaml

```diff
@@ -30,6 +30,7 @@ jobs:
       ARCH_NAME: ${{ matrix.test-group.arch }}
       LOGURU_LEVEL: INFO
       LD_LIBRARY_PATH: ${{ github.workspace }}/build/lib
+    environment: dev
```

Comment:
I don't think we need this anymore, no?

Reply:
I added this line in because the other yaml workflows, such as tg-unit-tests.yaml and t3000-unit-tests.yaml, have this line included, and I wasn't sure if omitting this line will cause any issues for the TGG unit tests workflow.

Reply:
pls delete

Reply:
we don't use environments anymore, and `TT_METAL_DEV` is a deprecated variable

Reply:
I deleted it.